css/style.css

```diff
@@ -0,0 +1,20 @@
+*,*:before,*:after{
```

Comment:
Don't use global selectors e.g. `*` or `*:before`. 
Specify which tags you need to style e.g. `body, .some-class:before, .some-class:after {...}`

Reply:
This is a nullification of common styles to avoid unnecessary code.

Reply:
Alright, then add spaces `*, *:before, *:after {`